### DISCUSSION TITLE
Is there any way to persist or export settings & configuration?

### DISCUSSION THREAD

**Discussion by vanillechai** [lobehub/lobehub#2067] [General]
(related to discussion #1275)

This concept of saving configs in the browser doesn't work for me. Not only do I have many devices with multiple browsers each (which would be annoying in itself), but most of these browsers erase their storage when you exit the browser.

So is there a way to persist or export settings & configuration other than the browser storage?

I'm using Docker specifically and feel comfortable with its internals.

If not, I suggest you mention this (browser persistence only) prominently in the README. For me, that would have been a knock-out criterion for not even starting to experiment with this (otherwise quite promising) chat.

**Comment by arvinxx** [comment]
refs to https://github.com/lobehub/lobe-chat/discussions/1768
